helm/thingsboard/templates/node.yaml

```diff
@@ -115,7 +115,7 @@ spec:
             value: {{ .value | quote }}
           {{- end }}
           - name: DEFAULT_INACTIVITY_TIMEOUT
-            value: {{ .Values.node.state.defaultInactivityTimeoutInSec | default 60 | quote }}
+            value: {{ .Values.node.state.defaultInactivityTimeoutInSec | default 600 | quote }}
```

Comment:
Have you verified that this works? Last time I tried to configure TB this way I could not make it work.